tests/e2e/specs/wcpay/merchant/merchant-disputes-view-details-via-order-notice.spec.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+/**
+ * Internal dependencies
+ */
+import { fillCardDetails, setupProductCheckout } from '../../../utils/payments';
+
+const { merchant, shopper } = require( '@woocommerce/e2e-utils' );
+
+describe( 'Disputes > View dispute details via disputed order notice', () => {
+	beforeAll( async () => {
+		await page.goto( config.get( 'url' ), { waitUntil: 'networkidle0' } );
+
+		// Place an order to dispute later
+		await setupProductCheckout(
+			config.get( 'addresses.customer.billing' )
+		);
+		const card = config.get( 'cards.disputed-fraudulent' );
+		await fillCardDetails( page, card );
+		await shopper.placeOrder();
+		await expect( page ).toMatch( 'Order received' );
+
+		// Get the order ID
+		const orderIdField = await page.$(
+			'.woocommerce-order-overview__order.order > strong'
+		);
+		const orderId = await orderIdField.evaluate( ( el ) => el.innerText );
+
+		await merchant.login();
+		await merchant.goToOrder( orderId );
+	} );
+
+	afterAll( async () => {
+		await merchant.logout();
+	} );
+
+	it( 'should click the disputed order notice and view the dispute details', async () => {
```

Comment:
naming nitpick … should the `should` focus on the expected behaviour, rather than the test steps? e.g. `it (should navigate to dispute details when disputed order notice button clicked`)`

(naming is hard, non blocking)

Reply:
Renamed to `should navigate to dispute details when disputed order notice button clicked` in c5a5ccb. Only a test name change.